src/worker.js

```diff
@@ -212,6 +212,25 @@ const queue = asyncQueue(({ id, data }, taskCallback) => {
 
             return options;
           },
+          getLogger: (name) => {
+            return {
+              error(message) {
+                console.error(`${name}: ${message}`);
+              },
+
+              warn(message) {
+                console.warn(`${name}: ${message}`);
+              },
+
+              log(message) {
+                console.log(`${name}: ${message}`);
+              },
+
+              debug(message) {
+                console.debug(`${name}: ${message}`);
+              },
+            };
+          },
```

Comment:
Ideally we should run webpack logger here, otherwise options from configuration for logging will not work

Reply:
ce90da5

Reply:
Can we map methods created logger, we can have more them `error`/`warn`/`log`/`debug`

Reply:
Just bumping this PR for some clarification.

> Can we map methods created logger, we can have more them `error`/`warn`/`log`/`debug`

@alexander-akait Do you mean to just loop through the list of methods instead of manually specifying them? e.g.

```javascript
const methods = ['error', 'warn', 'log', 'debug']];
const logger = {}

for (const key of methods ) {
    logger[key] = ...
}

return logger;
```

Reply:
@alexander-akait Nudging this PR to see what's blocking it?